api/governance/admin.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib import admin
+
+from .models import Proposal, ProposalVote
+
+
+@admin.register(Proposal)
+class ProposalAdmin(admin.ModelAdmin):
+    fields = ('proposed_by', 'votes', 'title', 'description',
+              'closed_at', 'created_at', 'updated_at')
```

Comment:
Consider adding the category tags and summary to the admin if other proposals are approved.

```suggestion
    fields = ('proposed_by', 'votes', 'title', 'description', 'summary',
              'category_tags', 'closed_at', 'created_at', 'updated_at')
```